main.go

```diff
@@ -54,6 +55,8 @@ const (
 	defaultLogLevel                = "info"
 	defaultDBLogLevel              = "warn"
 	commentProcessingDryRunDefault = true
+	defaultCRTimeRoundingFactor    = 1 * time.Hour
```

Comment:
WDYT of starting with 4 hours? I'm not sure an hour has much benefit, and it will increase costs